Source/WebKit/WebProcess/cocoa/PlaybackSessionManager.mm

```diff
@@ -577,6 +582,56 @@
     legibleMediaSelectionIndexChanged(contextId, model->legibleMediaSelectedIndex());
 }
 
+#if HAVE(PIP_SKIP_PREROLL)
+void PlaybackSessionManager::setMediaSessionAndRegisterAsObserver()
+{
+    if (!m_controlsManagerContextId) {
```

Comment:
May be able to simplify a tad with a `makeScopeExit` which you release for the once case m_mediaSession won't be set to null